projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -943,9 +943,7 @@ public function get_filtered_connection_data( $selected_post_id = null ) {
 					)
 					||
 					(
-						is_array( $connection )
```

Comment:
The point being that for the other checks to be true then $connection must be an array?

Reply:
Yes, it's already an array.